### PR TITLE
optimize worker taskProcessor logger.

### PR DESCRIPTION
**Pull request by yimaixinchen**
Adding loggers and return void if all processors in the Worker is null.if these processors in the Worker is not null ,adding loggers and continue to process . 

Before commit ,some processors has judged null, others processors has not judged null, the logger is the same situation.
After the commit,all Processors judge null and logger

### CODE REVIEW

File: dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/processor/DBTaskAckProcessor.java [hunk 1]
```diff
@@ -46,8 +46,10 @@ public void process(Channel channel, Command command) {
                 command.getBody(), DBTaskAckCommand.class);
 
         if (taskAckCommand == null){
+            logger.error("dBTask ACK request command is null");
```
Comment (caishunfeng) [inline dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/processor/DBTaskAckProcessor.java:49]: Can you add some identity here? such as processInstanceId or taskInstanceId, or name.

**Comment by yimaixinchen** [comment]
first,if these commands are null,the logger cannot get anything from these commands.so i think it can not log anything including taskInstanceId and name in the error log.
Second,it these commands are not null ,the cammand information can log in the info logger.And every command has taskInstanceId field.look at picture.
捕获
捕获3
so can it add the identity the logger like the picture?
捕获2

**Comment by yimaixinchen** [comment]
I think taskInstanceId  is  redundant in the info logger.because it is the field in the every command just like above picture.

**Review by caishunfeng — APPROVED** [review]
LGTM

**Pull request merged**